### PR TITLE
Replace foreman-tasks with dynflow service and other fixes

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: containers/roles/dynflow/files/start-dynflowd.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/sbin/dynflowd --foreman-root=/usr/share/foreman run
```
Comment (ekohl) [inline containers/roles/dynflow/files/start-dynflowd.sh]: Feels like a bit of overhead to use a bash script just to run it.

Comment (ehelms) [reply containers/roles/dynflow/files/start-dynflowd.sh]: What would you consider using instead?

Comment (ekohl) [reply containers/roles/dynflow/files/start-dynflowd.sh]: Isn't it an option to do this directly in the yaml file?

Comment (ehelms) [reply containers/roles/dynflow/files/start-dynflowd.sh]: It is, however, best practices I have seen is to keep the logic in a bash script even if its simplistic. I think to make the role more complete and to allow easier re-factoring of the start script (or if it becomes multiline)

**Comment by ehelms** [comment]
I'll look at #561

**Pull request merged**